views/advanced_usage.dt

```diff
@@ -55,4 +55,24 @@ block body
 	p For adding subpackages from the registry, execute <code> dub add &lt;packagename&gt;:&lt;subpackage&gt; </code>
 
 	p If you need to compile a subpackage only, use <code> dub build :&lt;subpackage&gt; </code>
-	
\ No newline at end of file
+
+	h2#have-version Conditional compilation according to dependency presence
+
+	p Projects can have additional special dependencies for specific configurations.
+
+	pre.code
+		|"configurations": [
+		|	{
+		|		"name": "myconfig",
+		|		"dependencies": {
+		|			"mydep": { "version": "*" }
+		|		}
+		|	}
+		|]
+
+	p DUB provides version identifier of dependencies for conditional compilation with <a href="https://dlang.org/spec/version.html#version">version conditions</a>. <code>have_&lt;dependency&gt;</code> version identifier can be used for conditional compilation.
```

Comment:
This is using the wrong casing, the `Have_` versions have an uppercase H.

Definition: https://github.com/dlang/dub/blob/059f7a2a16cf7a50a5c68dc295c796482116ff87/source/dub/generators/generator.d#L417

Also this should include how special characters are replaced (anything other than an identifier character `[a-zA-Z0-9_]` is replaced with `_`)

Reply:
Updated accordingly